src/reference/config/writing-platform-configs.rst

```diff
@@ -287,7 +287,8 @@ locations. The entire :term:`run directory` can be symlinked, as well as
 certain sub-directories.
 
 * ``run`` - the :term:`run directory` itself
-* ``log``
+* ``log`` (see :term:`workflow log directory`)
```

Comment:
The glossary appears to define "workflow log directory" as `log/scheduler`: https://cylc.github.io/cylc-doc/stable/html/glossary.html#term-workflow-log-directory

```suggestion
* ``log``
```